tests/resources/applications/gradle/liberty-gradle-test-app/src/main/java/test/maven/liberty/web/app/HelloServlet.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
```

Comment:
Let's stick to EPL 2.0 rather than dragging in different licenses.   Since it's IBM copyright (and Apache 2) we can relicense to EPL 2